student_software_developers.md

```diff
@@ -1,38 +1,53 @@
-# Student Software Developers
+# Student Software and UX Developers
 
 ## Program
 
 Our goal in this program is to fully integrate undergraduate developers into our team. These developers will be paired with members of DLS to initially build skills and then work issues to advance our software.
 
-## Student Employee Policy
-There is a library-wide [Student Employee Policy](https://pulstaff.princeton.edu/working-pul/library-student-employee-policy/) on the Connecting staff Intranet site.
+## Team Integration
 
-## Timesheets
+Student Developers will operate as a member of the DLS team, with the exception
+that their mentor acts as their direct supervisor. This means they will have a
+buddy from DLS as part of their onboarding, and will pair with various members
+of the team and have a weekly check-in with their mentor.
 
-Student Software Developers are paid biweekly. Timesheet and pay schedules are available on the [Payroll Cycles website](https://finance.princeton.edu/payroll-labor-accounting/payroll/payroll-basics/monthly-and-biweekly-payroll-cycles)
+During the summer students will attend all DLS team meetings and participate on
+the team slack channel. These communication outlets will allow for us to arrange
+pairing sessions as needed.
+
+During the school year students should continue to use the team slack channel to
+collaborate with DLS team members. If a student's work schedule allows them to
+attend the team stand-up they should do so. If not they should post a stand-up
+style check in to the team slack channel, and ask for a pair if needed. Students
+and mentors should continue to have a weekly 1:1 meeting.
+
+### Absence from Work
+
+If a student will miss their shift for any reason, they should post a message in the team slack channel saying that they won't be in. This ensures the team knows not to expect them in meetings, etc, even if their supervisor is also out. The student should follow up individually with their supervisor as to whether they will be using sick time or unpaid time.
+
+## Policies
```

Comment:
It's a shame that these Student Worker Policies are in three different places, but linking out seems like a good way to deal with it.

Reply:
Yeah, I am glad these external documents are pretty short at least.